api/envoy/config/core/v3/protocol.proto

```diff
@@ -662,6 +662,11 @@ message Http2ProtocolOptions {
 
   // Configure the maximum amount of metadata than can be handled per stream. Defaults to 1 MB.
   google.protobuf.UInt64Value max_metadata_size = 17;
+
+  // Disables encoding the headers using huffman encoding.
+  // This can be useful in cases where the cpu spent encoding the headers isn't
+  // worth the network bandwidth saved e.g. for localhost.
+  bool disable_huffman_encoding = 18;
```

Comment:
How about encoding this as `google.protobuf.BoolValue enable_huffman_encoding`, so that the data plane's default isn't necessarily baked into the API?  It also avoids a double negative -- i.e., instead of `if (!disable_huffman_encoding)`, we can say `if (enable_huffman_encoding)`.